### PR TITLE
CRIMAPP-506 capital check your answers for all capital questions

### PR DESCRIPTION
**Pull request by ripan**
## Description of change
Add "Check your answers" page

## Link to relevant ticket
https://dsdmoj.atlassian.net/browse/CRIMAPP-506

### Before changes:
Screenshot 2024-03-28 at 12 44 05

### After changes:
Screenshot 2024-03-28 at 12 42 14

Screenshot 2024-03-27 at 18 17 36

## How to manually test the feature
http://localhost:3000/applications/:id/steps/capital/check_your_answers_capital

### CODE REVIEW

**Review by timpeat — APPROVED** [review]
🚀 👏

File: app/presenters/summary/html_presenter.rb [hunk 1]
```diff
@@ -38,6 +38,15 @@ class HtmlPresenter
         supporting_evidence
         more_information
         legal_representative_details
+      ],
```
Comment (timpeat) [inline app/presenters/summary/html_presenter.rb:41]: I think we may need to introduce more structure to this configuration at some point but this is fine for now.

File: app/presenters/summary/sections/other_capital_details.rb [hunk 2]
```diff
@@ -10,7 +10,7 @@ def answers
           Components::ValueAnswer.new(
             :has_frozen_income_or_assets, capital.has_frozen_income_or_assets,
             change_path: edit_steps_capital_frozen_income_savings_assets_capital_path
-          ),
+          )
```
Comment (hiboabd) [inline app/presenters/summary/sections/other_capital_details.rb:13]: Are we not displaying the value of `has_no_other_assets` in the review application screen?

File: config/locales/en/steps.yml [hunk 3]
```diff
@@ -498,6 +498,10 @@ en:
             one: You have added %{count} asset
             other: You have added %{count} assets
       frozen_income_savings_assets_capital: *FROZEN_INCOME_ASSETS_PAGE_TEXT
+      answers:
+        edit:
+          page_title: Check your answers - Your client's capital - Apply for criminal legal aid - GOV.UK
```
Comment (hiboabd) [inline config/locales/en/steps.yml:503]: We don't usually have `- Apply for criminal legal aid - GOV.UK` on the end of our page titles 🤔 would be good to ask rachel why this has been added

Comment (timpeat) [reply config/locales/en/steps.yml:503]: It gets added automatically @hiboabd, so yes @ripan " - Apply for criminal legal..." needs removing here.

**Comment by ripan** [comment]
Description updated

**Review by hiboabd — APPROVED** [review]
LGTM thanks ripan!

**Pull request merged**